sources/2011-04-04-per-jessen.md

```diff
@@ -0,0 +1,182 @@
+
+
+**What especially motivates you to participate in the openSUSE project?**
+
+Nothing in particular. I’m just being loyal to my favourite distro.
+
+
+**What do you think was your most important contribution to the openSUSE project/community or what is the contribution that you’re most proud of?**
+
+I was quite pleased with myself when I succeeded in upgrading nasm (mostly to incorporate a patch I had submitted to the nasm-project a year ago), but it’s hardly an important contribution.
+
+
+**When do you usually spend time on the openSUSE project?**
+
+It varies, mostly during the week.
+
+
+**Three words to describe openSUSE? Or make up a proper slogan!**
+
+The green stuff!
+
+
+**What do you think is missing or underrated in the distribution or the project?**
+
+I think we need more leadership and better quality control. We have too many changes happening without proper review, consideration and planning.
+
+
+**What do you think the future holds for the openSUSE project?**
+
+No idea.
+
+
+**A person ask you why he/she should choose openSUSE instead of other distribution/OS. What would be your arguments to convince him/her to pick up openSUSE?**
+
+If someone asks me, it’s because they value my advice â€“ I hope that is enough to convince him or her.
+
+
+**Which members of the openSUSE community have you met in person?**
+
+None (as far as I know).
+
+
+**How many icons are currently on your desktop?**
+
+I don’t know, I don’t really use my desktop much. I have about 35-36 open windows.
+
+
+**What is the application you can’t live without? And why?**
+
+Openoffice â€“ because I do everything in openoffice. Asterisk – because it runs my office and home telephone system.
+
+
+**Which application or feature should be invented as soon as possible?**
+
+A unified communications device for use with DECT, Wifi, GSM etcetera. Anywhere, anytime. In essence, the Star Trek comms-badge.
+
+
+**Which is your preferred text editor? And why?**
+
+Today it’s kwrite and vi, but I really like(d) IBMs LPEX that was part of VisualAge C++.
+
+
+**Which famous person would you want to join the openSUSE community?**
+
+No-one.
+
+
+**Which computer related skills would you like to have?**
+
+I could probably do with acquiring some better perl skills, but it’s not a language I really like working with.
+
+
+**The Internet crashes for a whole week. How would you feel, what would you do?**
+
+Relax.
+
+
+**Which is your favorite movie scene?**
+
+From the Australian movie â€žThe Dishâ€œ when the local band plays the title-music from â€žHawaii Five-Oâ€œ thinking it is the American national anthem. 
+
+
+**Star Trek or Star Wars?**
+
+I like both actually.
+
+
+**What is your favorite food and drink?**
+
+KÃ¤sespÃ¤tzle mit RÃ¶stzwiebel. Ittinger KlosterbrÃ¤u. Erdinger Dunkelweiss.
```

Comment:
There are some issues with non-ASCII characters here and at some other places in this file. I suppose this should rather be "Käsepätzle mit Röstzwiebel. Ittinger Klosterbräu. Erdinger Dunkelweiss".